assets/css/index.css

```
@@ -1624,3 +1667,260 @@ input[type="range"]::-webkit-slider-thumb:hover {
+}
+
+.searchResultDuration {
+	position: absolute;
+	bottom: 4px;
+	right: 4px;
+	background-color: rgba(0, 0, 0, 0.85);
+	color: #fff;
+	padding: 2px 5px;
+	font-size: 0.75rem;
+	border-radius: 4px;
+	font-family: "JetBrains";
```

Stylelint (17.13.0)

[error] Expected no quotes around "JetBrains" (font-family-name-quotes)

(font-family-name-quotes)